simbad/rotsearch/dat_info.py

```diff
@@ -0,0 +1,25 @@
+class DatModelInfo(object):
```

Comment:
If we have many of these model classes, create a new subpackage --> `simbad.core`, create an abstract class with `DatModel` and then define stuff like `_as_dict()` ... this then allows us to inherit from that class and makes all the code much cleaner

Reply:
You could create an abstract class like this:

https://github.com/rigdenlab/conkit/blob/ed6dd7e14ffc9d6a7d26cae1117c54a9d34f4a97/conkit/misc/bandwidth.py#L49